.github/workflows/coverage.yaml

```diff
@@ -53,7 +53,10 @@ jobs:
           restore-keys: |
             coverage-${{ runner.os }}-cargo-${{ hashFiles('Cargo.lock') }}
             coverage-${{ runner.os }}-cargo-
-
+      - name: Create python virtual env
+        shell: bash
+        run: |
+          source ./dozer-tests/python_udf/virtualenv.sh
```

Comment:
Need Help

I want to create virtual env to run python udf test in it, but seems the script doesn't work. 

Do you have some thoughts?

Reply:
Looks fine to me. Don't know why it doesn't work.

Reply:
Yeah, also works in my local

Reply:
are each action steps executed in the same shell? I mean if the env is activated at "Create python virtual env" step, will be still active at "Run actions-rs/cargo@v1" step?

Reply:
Ah I think @mediuminvader got it here. Each step probablys runs in a different shell.

Reply:
Let me think how to fix it later :)

Reply:
done 😎